modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,171 @@
+
+[id="pipelines-new-features-1-9_{context}"]
+=== Pipelines
+
+* With this update, you can specify pipelines parameters and results in arrays and object dictionary forms.
+
+* This update provides support for Container Storage Interface (CSI) and projected volume for your workspace.
+
+* With this update, you can specify the `stdoutConfig` and `stderrConfig` parameters when defining pipeline steps. Defining these parameters helps to capture standard output and standard error associated with steps to local files.
+
+* With this update, you can add variables in the `steps[].onError` event handler, for example, `$(params.CONTINUE)`.
+
+* With this update, you can use the output from the `finally` task in the `PipelineResults` definition. For example, `$(finally.<pipelinetask-name>.result.<result-name>)`, where `<pipelinetask-name>` denotes the pipeline task name and `<result-name>` denotes the result name.
+
+* This update supports task-level resource requirements for a task run.
+
+* With this update, workspaces are propagated in embedded specifications of `PipelineRun` and `TaskRun` without any changes. Propagated parameters are now in beta, and enabled by default.
+
+* This update adds support for remote resolution, such as built-in git, cluster, bundle, and hub resolvers.
+
+
+[id="triggers-new-features-1-9_{context}"]
+=== Triggers
+
+* This update adds the `Interceptor` custom resource definition to define `NamespacedInterceptor`. You can use `NamespacedInterceptor` in the kind section of interceptors reference in triggers or in the `EventListener` specification.
+
+* With this update, you can use the `cloudEventURI` definition in your API.
+
+* With this update, you can configure the webhook port number when defining a trigger.
+
+* This update supports trigger `eventID` to be used as input to `TriggerBinding`.
+
+* This update supports validation and rotation of certificates for the `clusterinterceptor` custom resource definition. 
+
+
+[id="cli-new-features-1-9_{context}"]
+=== CLI
+
+* This update adds support for a pipeline run with minimal embedded status. 
+
+* This update supports showing annotations in the `describe` command.
+
+* This update supports showing pipeline, tasks, and finally timeout in the `pr describe` command.
+
+* This update adds flags to provide pipeline, tasks, and finally timeout in the `pipeline start` command.
+
+* This update adds the status of workspaces in the `describe` command of a task and pipeline.
+
+* This update adds a check for the maximum usage of bundle resources in the pipeline controller.
+
+* This update adds a new flag `--ignore-running-pipelinerun`, which ignores the deletion of `TaskRun` associated with `PipelineRun`.
+
+* With this update, you can use the parameters `qps` and `bursts` to configure the Tekton controller performance.
+
+* This update makes the Z shell (Zsh) completion feature usable without generating any files.
```

Comment:
that's humm not a feature but yeah sure :-D

Reply:
if this feature not needed, we can remove it